site/docs/v1/tech/core-concepts/applications.adoc

```diff
@@ -454,6 +454,9 @@ image::core-concepts/application-saml-authentication-request.png[Application SAM
 [field]#Require signature# [optional]#Optional#::
 When enabled, all unsigned requests will be rejected.
 
+[field]#Login hint parameter# [optional]#Optional# [since]#Available since 1.47.0#::
+The name of the login hint paramter provided by the service provider on an AuthnRequest. If this parameter is present, its value will be used to pre-populate the username field on the FusionAuth login form.
```

Comment:
```suggestion
The name of the login hint parameter provided by the service provider on an AuthnRequest. If this parameter is present, its value will be used to pre-populate the username field on the FusionAuth login form.
```

Reply:
I'm a bit confused about how this works. Say I configure the `loginHintParameter` to be `userHint`. That means in the AuthNRequest, I'll get something like `<AuthNRequest userHint="dan@example.com" ...>` and FusionAuth will prepopulate the login form with `dan@example.com`? Is that the case? If so, would be awesome to have that explanation in this doc somewhere.

Reply:
Added the sentence 

_Note that this setting names an HTTP query parameter, not an element in the SAML AuthnRequest XML._